panda.yml

```diff
@@ -1,5 +1,5 @@
 # Franka Emika Panda robot with hand
-urdf: "package://robowflex_resources/panda/urdf/panda_arm_hand.urdf.xacro"
-srdf: "package://robowflex_resources/panda/config/panda_arm_hand.srdf.xacro"
+urdf: "package://robowflex_resources/panda/panda_description/urdf/panda.urdf"
```

Comment:
This should follow the prior naming scheme where the urdf is in the "urdf" directory, not the "panda_description" directory.